src/main/java/net/dv8tion/jda/api/events/guild/update/GuildUpdateIconEvent.java

```diff
@@ -60,6 +61,22 @@ public String getOldIconUrl()
         return previous == null ? null : String.format(Guild.ICON_URL, guild.getId(), previous, previous.startsWith("a_") ? "gif" : "png");
     }
 
+    /**
+     * Returns an {@link ImageProxy} for this guild's old icon
+     * <p>
+     * <b>Note:</b> the old icon may not always be downloadable as it might have been removed from Discord
```

Comment:
```suggestion
     * Returns an {@link ImageProxy} for this guild's old icon.
     * <p>
     * <b>Note:</b> the old icon may not always be downloadable as it might have been removed from Discord.
```